### PR TITLE
feat(database): migrated to package

### PR DESCRIPTION
**Pull request by henningsieh**
## Release Notes

* **New Features**
  * Added end-to-end testing suite with multiple test execution modes (UI, debug, headed, reporting).
  * Introduced database seeding and schema management capabilities.

* **Chores**
  * Migrated package manager from Bun to pnpm across all scripts and documentation.
  * Centralized database schema management to improve maintainability.
  * Standardized internationalization translation key naming conventions.
  * Updated developer tooling configurations for improved testing and linting workflows.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: .github/prompts/plan-simplifyDistanceKmValidation.prompt.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ describe("Floating-Point Edge Cases", () => {
 Execute:
 
 ```bash
-bun run test src/__tests__/distance-validation.test.ts
+pnpm run test src/__tests__/distance-validation.test.ts
 ```
```
[the remark this answers is not in the document]
Comment (henningsieh) [reply .github/prompts/plan-simplifyDistanceKmValidation.prompt.md:255]: new convention in the turborepo is using pnpm over bun

File: packages/database/package.json [hunk 2]
```diff
@@ -12,14 +12,20 @@
...
   },
   "dependencies": {
     "@greendex/config": "workspace:*",
-    "@paralleldrive/cuid2": "^3.0.6",
-    "drizzle-orm": "^0.45.1"
+    "pg": "^8.17.2"
   },
   "devDependencies": {
+    "dotenv": "^17.2.3",
+    "drizzle-kit": "^0.31.8",
```
[the remark this answers is not in the document]
Comment (henningsieh) [reply packages/database/package.json:28]: wrong hint, you have not looked at the current file. the contents are different in the file!?!?!

**Comment by henningsieh** [comment]
@coderabbitai please do a complete PR review again. Take a special look at the "misleading initializeDb" issue - I refactored the database package service client to robust and clean code, I hope

**Comment by henningsieh** [comment]
@coderabbitai new learning: the whole project has migrated to a turborepo, using pnpm exclusively. Remember this! If there are still files refering to 'bun' that's an error or a missing rewrite then.
